app/page.module.css

```
@@ -0,0 +1,146 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  background-color: #1f79ff;
+  font-family: 'Arial', sans-serif;
```

Stylelint (17.9.0)

[error] Expected no quotes around "Arial" (font-family-name-quotes)

(font-family-name-quotes)